src/vilkaar/vilkaar.js

```diff
@@ -79,13 +99,45 @@ const vilkaar = [
     kode: 'FTRL_2_7_RIMELIGHETSVURDERING',
     term: 'Er det, etter en samlet vurdering, rimelig at søker får innvilget frivillig medlemskap?'
   },
+  {
+    kode: 'FTRL_2_7_FORSØRGET_FAMILIEMEDLEM',
+    term: 'Er søker forsørget og oppholder seg sammen med en person som er medlem etter § 2-7 første ledd?'
+  },
+  {
+    kode: 'FTRL_2_7_INGEN_SÆRLIGE_GRUNNER_TALER_IMOT',
+    term: 'Har du kommet frem til at det ikke er særlige grunner som taler imot at søker skal få innvilget frivillig medlemskap?'
+  },
   {
     kode: 'FTRL_2_7A_BOSATT_I_NORGE',
     term: 'Er søker bosatt i Norge?'
   },
   {
     kode: 'FTRL_2_7A_SKIP_UTENFOR_EØS',
     term: 'Er søker i arbeid på et skip registrert i en stat utenfor EØS-området?'
+  },
+  {
+    kode: 'FTRL_2_8_FORUTGÅENDE_TRYGDETID',
```

Comment:
Kommer det egen sak for fjerning av denne og tilhørende migrering? Ser den står som streket over i confluence.

Reply:
Det er det som er avtalt. Saken er ikke opprettet ennå. Sikkert et godt tidspunkt å gjøre det

Reply:
Margareth oppretter noe